test/extensions/transport_sockets/http_11_proxy/connect_test.cc

```diff
@@ -39,12 +39,18 @@ class Http11ConnectTest : public testing::TestWithParam<Network::Address::IpVers
 public:
   Http11ConnectTest() = default;
 
+  static inline const std::string kTestHostname = "test.example.com";
+  static inline const std::string kConnectRequestWithHostname = absl::StrCat(
+      "CONNECT ", kTestHostname, ":443 HTTP/1.1\r\nHost: ", kTestHostname, ":443\r\n\r\n");
```

Comment:
We deviate from the Google style guide when it comes to constant vars:
https://github.com/envoyproxy/envoy/blob/main/STYLE.md?plain=1#L62-L65

`TestHostname` will suffice here.

Reply:
fixed.